repobuild/collect_artifacts.sh

```diff
@@ -3,21 +3,26 @@
 dir=$(readlink -f $(dirname $0))
 out=$dir/artifacts
 
+# Use runner architecture to select artifact directories
+ARCH=$(uname -m)
+
 declare -A COPY_DIRS
-COPY_DIRS[rpm]="RPMS/x86_64 RPMS/noarch SRPMS"
+COPY_DIRS[rpm]="RPMS/${ARCH} RPMS/noarch SRPMS"
 COPY_DIRS[deb]="DEBS/**"
 
 declare -A POOLS
-POOLS[rpm]="x86_64/Packages"
+POOLS[rpm]="${ARCH}/Packages"
 POOLS[deb]="pool"
 
 # Make 'info' file with the artifacts details
 make_info () {
 BUILD_NUMBER='local_build'
 [ -n "$GITHUB_RUN_NUMBER" ] && BUILD_NUMBER=$GITHUB_RUN_NUMBER
-[ -n "$DRONE_BILD_NUMBER" ] && BUILD_NUMBER=$DRONE_BILD_NUMBER
+[ -n "$DRONE_BUILD_NUMBER" ] && BUILD_NUMBER=$DRONE_BUILD_NUMBER
```

Comment:
We haven't used Drone for years.  I bet this code can be removed.

Reply:
Yes, it was my change ) I've decided to fix this typo. It can be removed. But all our scrips, even in `elastio` and `devboxes` are still compatible with `drone.io` ))